esp32/README.md

```diff
@@ -91,6 +91,11 @@ $ git submodule init lib/berkeley-db-1.xx
 $ git submodule update
 ```
 
+You will also need to have PySerial installed if it isn't already.
+```bash
+$ pip install pyserial
+```
+
```

Comment:
I can double check this, but I'm pretty sure that I didn't have esptool installed when I built and flashed my board at the sprints - just pyserial. Not sure what's going on there, maybe the build is using the one in the ESP IDF? If esptool is useful later on then it might still make sense to install it.